Funcky.Async/Functional/RetryAsync.cs

```diff
@@ -1,9 +1,8 @@
-using Funcky.Async.Extensions;
 using Funcky.RetryPolicies;
 
-namespace Funcky.Async;
+namespace Funcky;
 
-public static partial class Functional
+public static partial class AsyncFunctional
```

Comment:
* Shouldn't this stay Functional? All Functions here should have a Postfix, which would avoid any conflict.

Reply:
You cannot have the same class in different Assemblies.